redbot/core/utils/chat_formatting.py

```diff
@@ -60,7 +60,7 @@ def question(text: str) -> str:
     return "\N{BLACK QUESTION MARK ORNAMENT} {}".format(text)
 
 
-def bold(text: str) -> str:
+def bold(text: str, escape_formatting: bool = True) -> str:
     """Get the given text in bold.
 
     Note: This escapes text prior to bolding.
```

Comment:
This needs an update, maybe:
```suggestion
    Note: By default, this function will escape ``text`` prior to emboldening.
```